strongbox-client/src/test/java/org/carlspring/strongbox/client/ArtifactResolverIntegrationTest.java

```diff
@@ -85,18 +84,8 @@ public MultiHttpClientConnThread(ArtifactResolver artifactResolver,
         @Override
         public final void run()
         {
-            try
-            {
-                artifactResolver.getResource(url);
-            }
-            catch (ArtifactTransportException e)
-            {
-                LOGGER.error(e.getMessage(), e);
-            }
-            catch (IOException e)
-            {
-                LOGGER.error(e.getMessage(), e);
-            }
+            CloseableRestResponse response = artifactResolver.get(url);
+            IOUtils.closeQuietly(response);
```

Comment:
I don't know, if you've come across this in the code, but we have a `ResourceCloser` class. For consistency, I would suggest using that instead, unless the `IOUtils` is better for us (in which case we'll have to do a general refactoring).

Reply:
Sure